app/models/katello/glue/provider.rb

```diff
@@ -95,11 +95,13 @@ def owner_upstream_update(upstream, _options)
         #ca_file = '/etc/candlepin/certs/upstream/subscription.rhn.stage.redhat.com.crt'
         ca_file = nil
 
-        capabilities = Resources::Candlepin::CandlepinPing.ping['managerCapabilities'].inject([]) do |result, element|
+        params = {}
+        params[:capabilities] = Resources::Candlepin::CandlepinPing.ping['managerCapabilities'].inject([]) do |result, element|
           result << {'name' => element}
         end
+        params[:facts] = {:distributor_version => 'sat-6.3'}
```

Comment:
I think future changes will likely involve code changes to handle the changes. It may be that the next iteration on the portal becomes more dynamic and code in katello is removed from the middle. My vote would be to hardcode in this release.